apps/ledger-live-mobile/src/locales/en/common.json

```diff
@@ -6844,6 +6844,9 @@
       "description": "Discover the best of web3 curated by Ledger"
     },
     "main": {
+      "manifestsList": {
+        "title": "Discover"
```

Comment:
Is it possible to put it in `web3hub.manifestsList` ? 
If not, could be a good idea to clarify a bit the difference btw  `web3hub.manifestsList ` and `web3hub.main.manifestsList `

Reply:
The difference is clear we have a manifestList component in web3hub that defaults to `web3hub.manifestsList` and we also use the manifestList in the main screen with a specific title so we also need `web3hub.main.manifestsList`

> Is it possible to put it in web3hub.manifestsList ?
That would be the wrong place to put it as it's only used in the main screen using the manifestList component and not in the component itself

Reply:
Both are components one is in main the other outside

If you want to be consistant with the actual folder hierarchy you would go for `web3hub.components.manifestList` and `web3hub.screens.main.components.manifestList` (you could probably do `web3hub.screens.main.manifestList` but then why not `web3hub.main.manifestList`)
But all this feels a bit convoluted just for strings that you can search